Makefile

```diff
@@ -0,0 +1,51 @@
+# Defining shell is necessary in order to modify PATH
+SHELL := sh
+export PATH := bin/:node_modules/.bin/:$(PATH)
+export NODE_OPTIONS := --trace-deprecation
+
+# On CI servers, use the `npm ci` installer to avoid introducing changes to the package-lock.json
+# On developer machines, prefer the generally more flexible `npm install`. 💪
```

Comment:
This comment should be updated since we use Yarn instead of npm.

Reply:
Fixerd.